sys/netbsd/fs.txt.const

```diff
@@ -0,0 +1,97 @@
+# Code generated by syz-sysgen. DO NOT EDIT.
```

Comment:
We now have 1 const file for all arches instead of file per arch. So please remove fs_amd64.const now. And the same for all other const files below.
Otherwise looks good.

Reply:
Done!